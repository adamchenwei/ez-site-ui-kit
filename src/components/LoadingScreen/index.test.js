import React from 'react';
import renderer from 'react-test-renderer';
import LoadingMask from './index';

describe('LoadingMask', () => {
  test('renders as expected', () => {
    const tree = renderer.create(<LoadingMask />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
