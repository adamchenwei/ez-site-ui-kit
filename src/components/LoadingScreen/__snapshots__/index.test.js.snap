// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingMask renders as expected 1`] = `
<section
  className="MaskingBackgroundContainer-goHeYq hxtXxm"
>
  <img
    className="LogoContainer-byHpFi dcqnxi"
    src="IMAGE_MOCK"
  />
  <section
    className="LoadingBarContainer-iZnhQC iRpnww"
  >
    <div
      className="Bar-gSSpUe eThZd"
    />
  </section>
</section>
`;
